TaskEngine/Tasks/DownloadMediaTask.cs

```diff
@@ -108,5 +115,75 @@ public async Task<Video> DownloadLocalPlaylist(Media media)
             }
             return video;
         }
+
+        // jason
+        public async Task<Video> DownloadBoxVideo(Media media)
+        {
+            var path = System.IO.Path.Combine(Directory.GetCurrentDirectory(), "refresh.json");
+            JObject refresh_json = JObject.Parse(File.ReadAllText(@path));
+            String access_token = (String)refresh_json.SelectToken("access_token");
+
+            while (true)
+            {
+                // send request using RestSharp
+                var client = new RestClient($"https://uofi.app.box.com/2.0/files/{media.Id}/content");
+                var request = new RestRequest(Method.GET);
+                request.AddHeader("cache-control", "no-cache");
+                request.AddHeader("Connection", "keep-alive");
+                request.AddHeader("Referer", $"https://uofi.app.box.com/2.0/files/{media.Id}/content");
+                request.AddHeader("Cookie", "box_visitor_id=5da4f447d00911.72030283");
+                request.AddHeader("Accept-Encoding", "gzip, deflate");
+                request.AddHeader("Postman-Token", "a02a657a-e03b-41b3-a747-6190a61f3bea,b759367e-377f-46a6-b96f-f3a78a984ad3");
+                request.AddHeader("Cache-Control", "no-cache");
+                request.AddHeader("Accept", "*/*");
+                request.AddHeader("User-Agent", "PostmanRuntime/7.18.0");
+                // TODO: figure out a way to refresh token
+                request.AddHeader("Authorization", $"Bearer {access_token}");
```

Comment:
Remove all the header that aren't needed, for example the ones to do with Postman